test/test_integration.py

```diff
@@ -1591,10 +1608,14 @@ def update_output(start_date, end_date):
         self.wait_for_text_to_equal('#date-picker-range-output', 'None - None')
 
         # updated only one date, callback shouldn't fire and output should be unchanged
+        dt_length = len(start_date.get_attribute('value'))
+        start_date.send_keys(dt_length * Keys.BACKSPACE)
         start_date.send_keys("1997-05-03")
         self.wait_for_text_to_equal('#date-picker-range-output', 'None - None')
 
         # updated both dates, callback should now fire and update output
+        dt_length = len(end_date.get_attribute('value'))
+        end_date.send_keys(dt_length * Keys.BACKSPACE)
```

Comment:
When used IRL the behaviour is ok (override existing text on click) -- but in Selenium this seems to be an issue now. Delete with backspace all characters..